dev/com.ibm.ws.cdi.beansxml_fat/fat/src/com/ibm/ws/cdi12/fat/tests/BeanDiscoveryModeNoneTest.java

```diff
@@ -56,19 +63,25 @@ public static Archive buildShrinkWrap() {
      * @see com.ibm.ws.fat.LoggingTest#getSharedServer()
      */
     @Override
-    protected ShrinkWrapSharedServer getSharedServer() {
-        return SHARED_SERVER;
+    protected SharedServer getSharedServer() {
+        return null;
     }
```

Comment:
If you're not using `getSharedServer()` then I guess you're not using any of the methods on `LoggingTest` either?

In that case you can probably stop extending `LoggingTest`.